client/src/Utils/dateHelper.js

```diff
@@ -0,0 +1,17 @@
+function dateHelper(date) {
```

Comment:
please rename function, it's not clear what exactly function helps with

Reply:
Renamed